application/src/main/java/org/togetherjava/tjbot/commands/help/ImplicitAskListener.java

```diff
@@ -181,24 +182,23 @@ private static MessageAction sendInitialMessage(ThreadChannel threadChannel,
             .setColor(HelpSystemHelper.AMBIENT_COLOR)
             .build();
 
-        Message threadMessage = new MessageBuilder(
-                """
-                        %s has a question about '**%s**' and will send the details now.
+        Message threadMessage = new MessageBuilder("""
+                %s has a question about '**%s**' and will send the details now.
 
-                        Please use `/change-help-category` to greatly increase the visibility of the question."""
-                    .formatted(author, title)).setEmbeds(embed).build();
+                Please use %s to greatly increase the visibility of the question."""
+            .formatted(author, title, MessageUtils.mentionSlashCommand(originalMessage.getGuild(),
+                    "change-help-category"))).setEmbeds(embed).build();
 
         return threadChannel.sendMessage(threadMessage);
     }
 
     private static MessageAction notifyUser(IMentionable threadChannel, Message message) {
         return message.getChannel()
-            .sendMessage(
-                    """
-                            %s Please use `/ask` to ask questions. Don't worry though, I created %s for you. \
-                            Please continue there, thanks."""
-                        .formatted(message.getAuthor().getAsMention(),
-                                threadChannel.getAsMention()));
+            .sendMessage("""
+                    %s Please use %s to ask questions. Don't worry though, I created %s for you. \
+                    Please continue there, thanks.""".formatted(message.getAuthor().getAsMention(),
+                    MessageUtils.mentionSlashCommand(message.getGuild(), "ask"),
```

Comment:
Id like to be able to introduce a compile-time dependency between this and the actual command. So that we dont have the raw string `"ask"` floating around here.

I.e. ideally there should be some form of linking this with the actual `AskCommand` which knows its name and has a `getName()` method.

To make this nice, we probably need a non-trivial rework and Im not sure if u can follow my thoughts, but:
* some sort of class that knows all commands and their names `CommandMentionService`
* all commands have to be registered there (for example by `BotCore`)
* the service is then created in `Features.java` like the other helpers
* people who need it, can then request it in their constructor params
* and then simply do `commandMentionService.mention(AskCommand.class)`

Reply:
Btw, sorry if this is too much. I definitely did not anticipate this when tagging the issue with "good first issue". So its totally okay if you want someone else to take over from here.

Reply:
Or we add a constant with the name to AskCommand? By far simplest, and I personally find adding a CommandMentionService sound overkill

Reply:
Yes. We discussed this in the server and Gyoo came up with the same idea 👍